### PR TITLE
Video processing

### PR DESCRIPTION
**Pull request by smorgsborgs**
getEncouragement (media) endpoint 
_Not_ production ready, but deployed and active on shared-app-dev and the kong dev instance. Naming is a bit funky given what the endpoint does. Need to add documentation for the dependency on AWS cloudformation template for media convert service, whose dynamodb table this interacts with.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+  
+#org: innovationit
+#app: rc-encourage-api
```
Comment (wilstaley) [inline serverless.yml:3]: Does it matter if these are commented out?

**Review by wilstaley — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (smorgsborgs) [reply serverless.yml:3]: They were my problem with deploying. I can take them out entirely if we want

Comment (wilstaley) [reply serverless.yml:3]: No worries!

**Review by Urbinmr — APPROVED** [review]

**Comment by smorgsborgs** [comment]
Will merge after demo per dependency on private cot registry

**Pull request merged**